DEPLOYMENT.md

```
@@ -112,11 +118,167 @@ The bucket stays private (`minio-init` runs `mc anonymous set none`); access is
+Langfuse instance (token/cost per Astra + `/deep` run). Run Langfuse once, on the infra server, via
+the `langfuse` profile of `infra/compose.dev.yaml` (Postgres + ClickHouse + Redis of its own; blob
+reuses the app MinIO):
+
+```bash
+# on the infra server, with LANGFUSE_* filled in infra/.env
+docker compose -f infra/compose.dev.yaml --profile langfuse up -d
+```
+
+Then point the prod agent at it by adding to the Dokploy **Environment** tab:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)